### ISSUE TITLE
Date Parsing improvement

### ISSUE THREAD

**Issue by g3d** [mak-it/saxlsx#1]
Could anyone please add such date format support?

http://d.pr/i/Xso0

**Comment by ebeigarts** [comment]
All date formats should be supported starting from v0.3.0. Are you using the latest version? Can you upload the file, if there are still problems with dates?

**Comment by g3d** [comment]
I use version 0.3.

File is here: https://www.dropbox.com/s/b07rhmjlb8w3fhs/students-1500.xlsx?dl=0
```ruby
::Saxlsx::Workbook.open(import_file_path) {|w| w.sheets.first.rows[1] }
=> ["JAMES",
 "SMITH",
 "JAMES",
 "Jack",
 "Male",
 35064.0,
 "01is2000@eduvo.com",
 "Sample Address 1321",
 "#644",
 "San Francisco",
 "California",
 82366.0,
 "United States",
 "+1 510 283-2064",
 "+1 415 283-9421",
 "English",
 "French",
 "German",
 "United States",
 "Germany",
 "United Kingdam",
 "2012-2013 Academic Year",
 nil,
 "Grade 10",
 "new.staff@eduvo.com",
 1123.0,
 "Pending",
 "Many",
 "I",
 "Many",
 "Many",
 38719.0]
```

**Comment by ebeigarts** [comment]
Are you sure you are using saxlsx? I just tested your example and it shows dates corectly:

```
[2] pry(main)> ap ::Saxlsx::Workbook.open(path) {|w| w.sheets.first.rows[1] }
[
    [ 0] "JAMES",
    [ 1] "SMITH",
    [ 2] "JAMES",
    [ 3] "Jack",
    [ 4] "Male",
    [ 5] Sun, 31 Dec 1995 00:00:00 +0000,
    [ 6] "01is2000@eduvo.com",
    [ 7] "Sample Address 1321",
    [ 8] "#644",
    [ 9] "San Francisco",
    [10] "California",
    [11] 82366.0,
    [12] "United States",
    [13] "+1 510 283-2064",
    [14] "+1 415 283-9421",
    [15] "English",
    [16] "French",
    [17] "German",
    [18] "United States",
    [19] "Germany",
    [20] "United Kingdam",
    [21] "2012-2013 Academic Year",
    [22] nil,
    [23] "Grade 10",
    [24] "new.staff@eduvo.com",
    [25] 1123.0,
    [26] "Pending",
    [27] "Many",
    [28] "I",
    [29] "Many",
    [30] "Many",
    [31] Mon, 02 Jan 2006 00:00:00 +0000
]
```

**Comment by ebeigarts** [comment]
Try printing `Saxlsx::VERSION`

**Comment by ebeigarts** [comment]
There is another problem, it returns the wrong date, there are two types of base dates in xlsx (1900 and 1904). Saxlsx currently supports only 1900, I will try to add 1904 support soon.

**Comment by g3d** [comment]
@ebeigarts how can I help you with that?

**Comment by ebeigarts** [comment]
I implemented 1904 date system in v1.0.0, try using v1.1.0

**Comment by g3d** [comment]
Got it. Thanks!
